frontend/src/components/Contexts/MissionControlContext.tsx

```diff
@@ -0,0 +1,79 @@
+import { createContext, useContext, useState, FC } from 'react'
+import { BackendAPICaller } from 'api/ApiCaller'
+import { Mission } from 'models/Mission'
+
+export enum ControlButton {
+    Pause,
+    Stop,
+    Resume,
+}
```

Comment:
ControlButton is already defined in MissionControlButtons.tsx, so we could consider to import it from there